media-video/showtime/showtime-49.0.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{11..14} )
+PYTHON_REQ_USE="threads(+)"
+
+inherit gnome.org gnome2-utils meson xdg python-single-r1 python-utils-r1
+
+DESCRIPTION="Video player for GNOME"
+HOMEPAGE="https://apps.gnome.org/Showtime/ https://gitlab.gnome.org/GNOME/showtime/"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="amd64 ~arm ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86"
+IUSE="+python"
+
+REQUIRED_USE="
+	${PYTHON_REQUIRED_USE}
+"
```

Comment:
Per the example for PYTHON_REQUIRED_USE at https://devmanual.gentoo.org/eclass-reference/python-single-r1.eclass/index.html#lbAG 

This should be nested in a USE flag conditional, if it is indeed conditional behind USE=python, even by default.